runner/src/server/services/upload/mockUploadService.ts

```diff
@@ -0,0 +1,32 @@
+import { UploadService } from "./uploadService";
+
+export class MockUploadService extends UploadService {
+  async uploadDocuments(locations: any[]) {
+    console.log("The locations: ", locations);
+    const shouldFail =
+      locations.filter((location) => location.hapi.filename === "bad-file.png")
+        .length > 0;
+    console.log("Should fail: ", shouldFail);
+    const responseData = shouldFail
+      ? {
+          res: {
+            statusCode: 201,
+            headers: {
+              location: "https://document-upload-endpoint",
+            },
+          },
+          payload: "imageQualityWarning",
+        }
+      : {
+          res: {
+            statusCode: 201,
+            headers: {
+              location: "https://document-endpoint.com",
+            },
+          },
+          payload: undefined,
+        };
```

Comment:
```suggestion
    const shouldFailOCR =
      locations.filter((location) => location.hapi.filename === "bad-file.png")
        .length > 0;

    const responseData = {
          res: {
            statusCode: 201,
            headers: {
              location: "https://document-upload-endpoint",
            },
          },
          payload: shouldFailOCR ? "imageQualityWarning" : undefined,
        }
      
```

Reply:
Although in perhaps in future we should allow the other failure states (like virus) be mocked. In which case we'd do an object lookup rather than ternary. something for another time though.